app/src/main/java/com/islaharper/dawnofandroid/presentation/screens/splash/SplashScreen.kt

```diff
@@ -57,7 +59,7 @@ private fun Splash(composition: LottieComposition?, progress: Float) {
     Box(
         modifier = Modifier
             .fillMaxSize()
-            .background(MaterialTheme.colorScheme.secondary),
+            .background(Teal40),
```

Comment:
`Teal40` is in the theme colors as `tertiary`. Won't using a theme color here help the splash screen adapt to dark mode?

Reply:
I did have it using tertiary and onTertiary etc, but unhappy with the way it looks, have created an issue, #8 , to finalise the colourScheme at a later point.